core/src/main/java/org/springframework/security/authentication/AbstractUserDetailsReactiveAuthenticationManager.java

```diff
@@ -22,6 +22,8 @@
 import reactor.core.scheduler.Scheduler;
 import reactor.core.scheduler.Schedulers;
 
+import org.springframework.context.MessageSource;
```

Comment:
Please make sure that each file uses the most recent copyright year. In this case, that would mean changing `2002-2019` to `2002-2020`.